containers/blockchain/shopBackend/app.js

```diff
@@ -60,6 +62,22 @@ const utils = require('./utils/util');
       utils.queueRequest(params, executeEvent);
     });
   });
+
+  // pass params to iot dashboard
+  function iotDashboard(data) {
+    var options = {
+      method: 'GET',
```

Comment:
I suggest using `POST` instead of `GET` and send the block data in the request body. You might encounter some problems in `JSON.stringify(data)`. I think its result wouldn't be URL encoded?

Reply:
It's been tested in a standalone app and submitted yesterday. Let's keep it as get while data is transmitted, post will require headers and re-structuring.